### PR TITLE
Adds 2022-Q1 Kirel Podium audio

### PR DESCRIPTION
**Pull request by ZarothYe**
PRP changes by Hazado.
Audio recordings directed by DapperAndy.
Audio post-processing by neurovac.

English voice: Tyion
French voice: skyisblu
German voice: Taliana (IC Name: Anna Windisch)
Italian voice: Fujhoo

### CODE REVIEW

**Comment by Hoikas** [comment]
Converting to draft because an audio file is pending.

File: compiled/sfx/krelFanQ12022_eng.sub [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+00:01:23,400 --> 00:01:28,400
+Tyion: Uh, check in with the community for any one-off changes to that.
+
+9
+00:01:29,400 --> 00:01:43,600
+Tyion: Secondly, the Guild of Messengers have a very well-maintained event calendar that covers pretty much everything from lessons on the D'ni language, story nights, to live music and meditation sessions.
+
+10
+00:01:44,000 --> 00:01:53,800
+Tyion: And you can find all that at the Guild website, uh, www.guildofmessengers.org/calender.
```
Comment (Hoikas) [inline compiled/sfx/krelFanQ12022_eng.sub:39]:
```suggestion
Tyion: And you can find all that at the Guild website, uh, www.guildofmessengers.org/calendar.
```

File: compiled/sfx/krelFanQ12022_eng.sub [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+00:00:28,400 --> 00:00:31,800
+Tyion: Right. Kicking off with some general admin.
+
+5
+00:00:32,200 --> 00:00:46,400
+Tyion: We've had some questions from explorers wondering how best to stay in the loop with what's going on down here, and we figured it was best to give you all a summary of what we have to work with right now.
+
+6
+00:00:47,800 --> 00:01:00,600
+Tyion: Firstly, the All Guilds Meeting takes place on the first Saturday of every month here in Kirel at approximately 13:00 KI time, barring of course holidays or Mysterium.
```
Comment (Hoikas) [inline compiled/sfx/krelFanQ12022_eng.sub:23]:
```suggestion
Tyion: Firstly, the All Guilds Meeting takes place on the first Saturday of every month here in Kirel at approximately 1300 KI time, barring of course holidays or Mysterium.
```

Maybe pedantic, but colons aren't used when time is read out as, eg, "thirteen hundred hours" (source: work).

File: compiled/sfx/krelFanQ42021_ger.sub [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+00:04:54,400 --> 00:04:59,600
+Taliana: Wer sich für das Projekt interessiert, sollte die diversen Social Media Seiten des Projektleiters Doobes im Auge behalten.
+
+32
+00:05:00,000 --> 00:05:02,400
+Taliana: Da kann man den Fortschritt anhand der Meldungen nachverfolgen.
+
+33
+00:05:03,600 --> 00:05:08,200
+Taliana: Ich denke, das war's, oder? Perfekt! Hoffen wir, dass es klappt!
```
Comment (Hoikas) [inline compiled/sfx/krelFanQ42021_ger.sub:131]: Newline at EOF please

Comment (ZarothYe) [reply compiled/sfx/krelFanQ42021_ger.sub:131]: Dangit, I thought I checked all the files for that. Must have missed this one. Thanks!

File: compiled/sfx/krelFanQ12022_eng.sub (hunk 2 above)
Comment (ZarothYe) [reply compiled/sfx/krelFanQ12022_eng.sub:23]: I did wonder about that when I saw it, but forgot to look it up to check. Good looking out!

**Comment by Hoikas** [comment]
Are we still "waiting on final audio processing?"

**Comment by ZarothYe** [comment]
> Are we still "waiting on final audio processing?"

No, as of https://github.com/H-uru/moul-assets/pull/113/commits/c3dd916766ee2ddfb11321fbd30fedc33b74a19b the final audio versions from neurovac are in this PR. I'll edit my original description.

**Pull request merged**